packages/feathers/src/declarations.ts

```diff
@@ -310,8 +318,14 @@ export interface HookContext<A = Application, S = any> extends BaseHookContext<S
   /**
    * A writeable, optional property that allows to override the standard HTTP status
    * code that should be returned.
+   *
+   * @deprecated
```

Comment:
I believe if we add a description here like `Use http.statusCode instead` it will show up accordingly in editors that support it.

Reply:
Done.